tests/python/include/folly/tracing/StaticTracepoint-ELF.h

```diff
@@ -18,8 +18,12 @@
 
 // Default constraint for the probe arguments as operands.
```

Comment:
Well, this file is not `StaticTracepoint-ELF.h` per say:) It's just x86 and PowerPC. How about we make a `StaticTracepoint-Impl.h` and have x86 and Power PC has their own header and define the `FOLLY_SDT_ARG_CONSTRAINT` and `FOLLY_SDT_ARGTMPL` there?
Also, we we are done, it would be great if you could contribute this change to [folly](https://github.com/facebook/folly) as well!

Reply:
I modeled this after SystemTap's `sys/sdt.h` where `STAP_SDT_ARG_CONSTRAINT` is set to `nor` for all architectures. However, for use with `bcc` running on `powerpc64`, we would prefer using `nQr` instead. So, would it be okay if I just removed `#elif defined(__x86_64__) || defined(__i386__)` from line 23 of `StaticTracepoint-ELF.h` and replace it with an `#else`?

As for contributing the changes to [folly](https://github.com/facebook/folly), I will definitely look into it.

Reply:
@palmtenor ping